wagtail/admin/templates/wagtailadmin/shared/editing_sessions/list.html

```diff
@@ -106,12 +106,12 @@
 {% if sessions.0.revision_id %}
     <template data-controller="w-teleport" data-w-teleport-target-value="#title-text-w-overwrite-changes-dialog" data-w-teleport-reset-value="true">
         {% if current_session.user == sessions.0.user %}
-            {% blocktranslate trimmed with model_name=content_type.name asvar someone_has_saved_message %}
-                You have saved a newer version of this {{ model_name }} in another session
+            {% blocktranslate trimmed asvar someone_has_saved_message %}
+                You have saved a newer version in another window
```

Comment:
```suggestion
                You saved a new version in another window
```

I wonder if we should reuse the exact same wording to simplify further?

Reply:
Yeah, I don't see a reason why it needs to be different to the one in the popup. I've amended the commit to simplify this as well. Thanks!